src/testdir/test_matchfuzzy.vim

```diff
@@ -300,11 +300,20 @@ func Test_matchfuzzy_initialized()
 
   let buf = RunVimInTerminal('-u NONE -X -Z', {})
   call term_sendkeys(buf, ":source XTest_matchfuzzy\n")
-  call TermWait(buf, 2000)
+  " Use term_wait directly rather than the TermWait wrapper; otherwise,
+  " retries become very slow.
+  call term_wait(buf, 2000)
 
   let job = term_getjob(buf)
   if job_status(job) == "run"
     call job_stop(job, "int")
+    " The search might or might not have been completed. If the search is
+    " finished and Vim receives a SIGINT, then that will trigger a message
+    " next time Vim is active:
+    "   Type  :qa  and press <Enter> to exit Vim
+    " If we do not send something here to trigger displaying the message, before
+    " TermWait(), then the exit sequence sent afterward does not work.
+    call term_sendkeys(buf, "\<C-O>")
```

Comment:
Hm, I am wondering if sending `<nop>` or `<ignore>` would be clearer?

Reply:
Those do work, but from experimentation, if I do this:
```
call term_sendkeys(2, "\<Nop>")
```
...then vim sends the five characters `<`, `N`, `o`, `p`, `>`, rather than any special code. Likewise with `\<nop>` and `\<ignore>` I think this is because these are not actual keycodes as listed in the help for `keycodes`, but I am not very familiar with them.

For the purpose of this test, sending various things seems to work (even `foo`), but I was aiming for something innocuous. If you have a recommendation, please let me know.